media/js/firefox/new/experiment-cliqz-funnelcakes.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    // initialize experiment - performs env check and does geo-lookup
+    Mozilla.TCFCGeoExp.init({
+        countryCode: 'de',
+        experimentConfig: {
+            id: 'experiment_cliqz_funnelcake',
+            variations: {
+                'f=119': 25,
+                'f=120': 25,
+                'f=121': 25,
+                'f=122': 25
```

Comment:
Just verifying these percentages are set (didn't see anything specific in the bug), or are known to need updating prior to merge.

Reply:
Yes these are final (confirmed in the solidarity that is Slack, alas).